scene/gui/option_button.cpp

```diff
@@ -75,6 +75,12 @@ void OptionButton::_notification(int p_what) {
 	}
 }
 
+void OptionButton::_visibility_changed() {
```

Comment:
This is ok, but should use NOTIFICATION_VISIBILITY_CHANGED in _notification callback.

Reply:
Agree. It makes the fix simpler.